src/api/uploads/FolderUploadNode.js

```diff
@@ -147,7 +147,7 @@ class FolderUploadNode {
             progress: 100,
         };
 
-        if (errorEncountered) {
+        if (errorEncountered && errorCode !== ERROR_CODE_ITEM_NAME_IN_USE) {
```

Comment:
since we do not want to treat "name in use" as an error, we should not set the values in that case. this change should not happen here; instead we should evaluate the try-catch ti set the scope variables appropriately.